themes/default/content/blog/mlops-huggingface-llm-aws-sagemaker-python/index.md

```diff
@@ -0,0 +1,131 @@
+---
+date: 2023-09-08
+title: "Deploy a Huggingface Model on AWS SageMaker with Pulumi Python IaC"
+allow_long_title: true
+meta_desc: "Guided short tutorial on starting a Pulumi Infrastructure as Code project to deploy Huggingface LLM Models on Amazon Web Services SageMaker Machine Learning Platform"
+meta_image: "meta.png"
+authors:
+    - kat-morgan
+tags:
+    - ai
+    - ml
+    - iac
+    - aws
+    - llm
+    - vllm
+    - mlops
+    - llama
+    - llama2
+    - python
+    - sagemaker
+    - huggingface
+---
+
+Welcome to another post in the Pulumi + Python MLOps #MLOpsChallenge series!
+
+In this short tutorial we will deploy a publicly available Huggingface model on Amazon AWS SageMaker, and test a prompt using a short python script.
```

Comment:
I believe their official title is Amazon Sagemaker, but pick one

Reply:
Fixed throughout in 46fb9a7